src/events/ready.mts

```diff
@@ -7,9 +12,36 @@ export default {
     once: true,
     // Define execution function which in this case is just print out bot user tag
     execute(client: Client) {
-        if (client.user === null) {
+        if (!client.user) {
             throw new Error("client.user is null");
         }
         console.log(`Ready! Logged in as ${client.user.tag}`);
+
+        // Schedule weekly audit log saving:
+        // Saturday @ 11:59 PM
+        cron.schedule("59 23 * * 6", async () => {
```

Comment:
Just do Sunday @ Midnight
```suggestion
        cron.schedule("0 0 * * 0", async () => {
```

Reply:
I can do this, just gotta change the dates in the file name to start at the previous Sunday and end at the Saturday right before the logs are saved. That's if we ignore the fact that it technically goes one minute over the time period.

Reply:
Going to keep the end time at 11:59 Saturday since it makes more sense for the dates.